jadx-plugins/jadx-xapk-input/src/main/java/jadx/plugins/input/xapk/XapkInputPlugin.kt

```diff
@@ -8,7 +8,7 @@ import jadx.plugins.input.dex.DexInputPlugin
 class XapkInputPlugin : JadxPlugin {
 	private val codeInput = XapkCustomCodeInput(this)
 	private val resourcesLoader = XapkCustomResourcesLoader()
-	internal var dexInputPlugin: DexInputPlugin? = null
+	internal var dexInputPlugin = DexInputPlugin()
```

Comment:
Why create a new instance of the DexInputPlugin if we can use the one already available?

Reply:
Well, until jadx have some API to do this search nicely, better not to rely on such hacky code :rofl: